src/datasources/results/constants/ResultsQueryBuilder.constants.ts

```diff
@@ -0,0 +1,183 @@
+import { QueryBuilderOperations } from 'core/query-builder.constants';
+import { QBField } from '../types/QueryResults.types';
+
+export enum ResultsQueryBuilderFieldNames {
+  PART_NUMBER = 'PartNumber',
+  PROGRAM_NAME = 'ProgramName',
+  SERIAL_NUMBER = 'SerialNumber',
+  KEYWORDS = 'Keywords',
+  PROPERTIES = 'Properties',
+  OPERATOR = 'Operator',
+  STARTED_AT = 'StartedAt',
+  UPDATED_AT = 'UpdatedAt',
+  SYSTEM_ID = 'SystemId',
+  WORKSPACE = 'Workspace',
+  STATUS = 'Status',
+  HOSTNAME = 'HostName',
+}
+
+export const ResultsQueryBuilderFields: Record<string, QBField> = {
+  PARTNUMBER: {
+    label: 'Part number',
+    dataField: ResultsQueryBuilderFieldNames.PART_NUMBER,
+    filterOperations: [
+      QueryBuilderOperations.EQUALS.name,
+      QueryBuilderOperations.DOES_NOT_EQUAL.name,
+      QueryBuilderOperations.STARTS_WITH.name,
+      QueryBuilderOperations.ENDS_WITH.name,
+      QueryBuilderOperations.CONTAINS.name,
+      QueryBuilderOperations.DOES_NOT_CONTAIN.name,
+      QueryBuilderOperations.IS_BLANK.name,
+      QueryBuilderOperations.IS_NOT_BLANK.name,
+    ],
+    lookup: {
+      dataSource: [],
+    },
+  },
+  PROGRAMNAME: {
+    label: 'Test Program',
```

Comment:
```suggestion
    label: 'Test program',
```

Use sentence case for user visible property names similar to other places in SLE